scripts/linux_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash +x
+
+BINARY_URL="https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux"
+SCRIPT="Linux"
+
+green () { echo -e "\033[01;32m$1:\033[0m $2"; }
+
+warn () { echo -e "\033[01;33m[$SCRIPT]WARNING:\033[0m $1"; }
+
+fatal () { echo -e "\033[01;31m[$SCRIPT]ERROR:\033[0m $1" >&2; exit 1; }
+
+architecture_version () {
+    case $(uname -m) in
+        x86_64) echo "amd64" ;;
+        # arm64)  echo "arm64" ;; ARM64 is unsupported on Linux for now
+        *)      fatal "$(uname -m) unsupported." ;;
+    esac
+}
+
+check_supported_distro () {
+    grep '^ID.*=.*ubuntu' /etc/os-release &> /dev/null || fatal "Aborting: You are using an unsupported distribution."
```

Comment:
any reason to support only Ubuntu ?? maybe rely on ID_LIKE key (from /etc/os-release)  will be better? to accept all Debian like distros.

Reply:
![image](https://user-images.githubusercontent.com/34547263/197177152-45e3a1fe-0f73-4876-bd8c-1909ed8068f1.png)

Reply:
fyi a screenshot of the os-release file from my vps running on debian.
![image](https://user-images.githubusercontent.com/34547263/197247745-6f94c2c1-41bf-4f5b-91ef-8de0922f1ddc.png)

Reply:
>any reason to support only Ubuntu ?? 

It has been decided to only support Ubuntu for now since it's one of the most used and known distribution. We are planning to had more supported distros following user demands. 

> maybe rely on ID_LIKE key (from /etc/os-release) will be better? to accept all Debian like distros
In fact, I am relying on `ID` and `ID_LIKE`. My regex `'^ID.*=.*ubuntu'` checks for ubuntu in both `ID` and `ID_LIKE`. Which allows us to support Ubuntu based distributions such as PopOS, Linux Mint, Zorin OS...

Reply:
fair enough :)